src/cmd/services/m3query/config/config.go

```diff
@@ -52,6 +52,9 @@ type Configuration struct {
 	// RPC is the RPC configuration.
 	RPC *RPCConfiguration `yaml:"rpc"`
 
+	// Backend is the backend store for query service. We currently support grpc and m3db (default).
+	Backend string `yaml:"backend"`
```

Comment:
nit: make an enumerated type for this

```

type BackendStorageType string 

const (
  GRPCStorageType BackendStorageType = "grpc"
  M3DBStorageType BackendStorageType = "m3db"
)

type Configuration struct {

... 
// Backend ...
Backend BackendStorageType ...
```

Probably need to add validation too.